drivers/pwm/Kconfig.nrfx

```diff
@@ -20,3 +20,8 @@ config PWM_NRFX
 	select PINCTRL
 	help
 	  Enable support for nrfx Hardware PWM driver for nRF52 MCU series.
+
+config PWM_NRFX_INIT_PRIORITY
+	int
```

Comment:
```suggestion
	int "NRFX PWM init priority"
```

Reply:
Do we want it to be user-assignable? In my opinion, `CONFIG_PWM_INIT_PRIORITY` can still be set and used if the PWM instance is not 120, in which case priority needs to be 99 to match clock device